bigtable/tests/system.py

```diff
@@ -449,6 +451,35 @@ def test_drop_by_prefix_table(self):
 
         self.assertEqual(expected_rows_count, read_rows_count)
 
+    def test_yield_rows_with_row_set(self):
+        row_keys = [
+            b'row_key_1', b'row_key_2', b'row_key_3', b'row_key_4',
+            b'row_key_5', b'row_key_6', b'row_key_7', b'row_key_8',
+            b'row_key_9']
+
+        for row_key in row_keys:
+            row = self._table.row(row_key)
+            row.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL1)
+            row.commit()
+            self.rows_to_delete.append(row)
+
+        row_set = RowSet()
+        row_set.add_row_range(RowRange(start_key=b'row_key_3',
+                                       end_key=b'row_key_7'))
+        row_set.add_row_key(b'row_key_1')
```

Comment:
This `row_set` object is not used in the rest of the system test (it should be passed to `yield_rows()`, IIUC).

Reply:
This was fixed.